libcontainer/configs/cgroup_linux.go

```diff
@@ -84,6 +84,10 @@ type Resources struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	//nolint:revive
```

Comment:
Maybe instead of ignoring revive warning, call it `CPUIdle` (hopefully we'll rename the rest of `Cpu*` fields at some point).